graylog2-web-interface/src/components/graylog/Navbar.jsx

```diff
@@ -0,0 +1,260 @@
+// eslint-disable-next-line no-restricted-imports
+import { Navbar as BootstrapNavbar } from 'react-bootstrap';
+import styled from 'styled-components';
+import { darken, lighten } from 'polished';
+
+import teinte from 'theme/teinte';
+
+const Navbar = styled(BootstrapNavbar)`
+  &.navbar-default {
+    background-color: ${teinte.secondary.due};
+    border-color: ${darken(0.065, teinte.secondary.due)};
+
+    .navbar-brand {
+      color: ${teinte.tertiary.uno};
+      &:hover,
+      &:focus {
+        color: ${darken(0.1, teinte.tertiary.uno)};
+        background-color: transparent;
+      }
+    }
+
+    .navbar-text {
+      color: ${teinte.primary.tre};
+    }
+
+    .navbar-nav {
+      > li > a {
+        color: ${teinte.tertiary.uno};
+
+        &:hover,
+        &:focus {
+          color: ${darken(0.25, teinte.tertiary.uno)};
+          background-color: transparent;
+        }
+      }
+      > .active > a {
+        &,
+        &:hover,
+        &:focus {
+          color: ${darken(0.15, teinte.tertiary.uno)};
+          background-color: ${darken(0.065, teinte.secondary.due)};
+        }
+      }
+      > .disabled > a {
+        &,
+        &:hover,
+        &:focus {
+          color: ${darken(0.25, teinte.secondary.tre)};
+          background-color: transparent;
+        }
+      }
+
+      > .open > a {
+        &,
+        &:hover,
+        &:focus {
+          color: ${darken(0.15, teinte.tertiary.uno)};
+          background-color: ${darken(0.065, teinte.secondary.due)};
+        }
+      }
+
+      @media (max-width: @grid-float-breakpoint-max) {
```

Comment:
I wish that would work, but as far as I know, this needs to be a fix width.

Reply:
It was just a missed variable replacement, thanks for spotting it